src/main/target/common_fc_post.h

```diff
@@ -29,6 +29,10 @@
 #undef USE_SERVOS
 #endif
 
+#ifndef USE_DSHOT
+#undef USE_ESC_SENSOR
```

Comment:
`#undef USE_ESC_SENSOR` should also be guarded by `#ifdef USE_ESC_SENSOR` in my opinion

Reply:
@DieHertz: Why is that? `#undef` for a macro that is not defined has no effect, so checking for definedness is somewhat redundant.